src/Tables.php

```diff
@@ -31,15 +26,19 @@ protected function get_pdo() {
 	 * @return void
 	 */
 	public function run( $assoc_args = [] ) {
-		$pdo = $this->get_pdo();
-
 		// Get all tables
-		$stmt   = $pdo->query( "SELECT name FROM sqlite_master WHERE type='table' AND name NOT LIKE 'sqlite_%'" );
-		$tables = $stmt->fetchAll( PDO::FETCH_COLUMN );
+		$tables = array();
+		foreach ( $this->driver->query( 'SHOW TABLES' ) as $row ) {
+			$tables[] = array_values( (array) $row )[0];
+		}
 
-		// Remove system tables
-		$tables_to_exclude = array( '_mysql_data_types_cache' );
-		$tables            = array_diff( $tables, $tables_to_exclude );
+		// With the legacy driver, we need to exclude system tables
+		// and make sure the table names are alphabetically sorted.
+		if ( $this->driver instanceof WP_SQLite_Translator ) {
+			$tables_to_exclude = array( '_mysql_data_types_cache', 'sqlite_sequence' );
```

Comment:
Should we also exclude `_wp_sqlite_mysql_information_schema_tables` , `_wp_sqlite_mysql_information_schema_columns`, `_wp_sqlite_mysql_information_schema_statistics`, and `_wp_sqlite_mysql_information_schema_table_constraints`?

Reply:
Fixed in https://github.com/Automattic/wp-cli-sqlite-command/pull/10/commits/0ac5622b4419d2689d593e79903c11daa1f74cc2.